dashboards/ec2/docs/ec2_application_load_balancer_detail.md

```diff
@@ -0,0 +1,3 @@
+This dashboard answers the following questions:
+
+- What relationships does this load balancer have with other resources?
```

Comment:
```suggestion
- What relationships does the load balancer have with other resources?
```

Small difference, but I think we usually refer to resources like `the instance`, `the bucket`, instead of `this instance`, `this bucket`, etc., in all of our existing dashboard docs.